ai/src/main/java/com/alibaba/nacos/ai/service/McpServerTransformService.java

```
@@ -0,0 +1,666 @@
+        StringBuilder command = new StringBuilder();
+        
+        switch (registryName.toLowerCase()) {
+            case "npm":
+                command.append("npx ").append(packageName);
+                break;
+            case "pypi":
+                command.append("python -m ").append(packageName);
+                break;
+            default:
+                command.append(packageName);
+                break;
+        }
+        
+        // Add package arguments if available
+        if (mcpPackage.getPackageArguments() != null) {
+            for (Argument arg : mcpPackage.getPackageArguments()) {
+                String argValue = extractArgumentValue(arg);
+                if (StringUtils.isNotBlank(argValue)) {
+                    command.append(" ").append(argValue);
+                }
+            }
+        }
+        
+        return command.toString();
+    }
+    
+    /**
+     * Extract argument value from polymorphic Argument interface.
+     * 
+     * @param arg Argument instance (either NamedArgument or PositionalArgument)
+     * @return argument value
+     */
+    private String extractArgumentValue(Argument arg) {
+        if (arg instanceof NamedArgument) {
+            NamedArgument namedArg = (NamedArgument) arg;
+            return namedArg.getValue();
+        } else if (arg instanceof PositionalArgument) {
+            PositionalArgument positionalArg = (PositionalArgument) arg;
+            return positionalArg.getValue();
+        }
+        return null;
+    }
+    
+    /**
+     * Parse URL data to MCP servers.
+     *
+     * @param urlData URL data string
+     * @return list of MCP server detail info
+     * @throws Exception if parsing fails
+     */
+    private List<McpServerDetailInfo> parseUrlData(String urlData) throws Exception {
+        // For URL import, we would typically fetch the data from the URL
+        // For now, we'll treat it as JSON data
```

Should we throw Unsupported exception for now？And Add todo？

---

@KomachiSion PTAL. [6b4c79a](https://github.com/alibaba/nacos/pull/13667/commits/6b4c79ac13da2eb3f75ca04915d6843243728e0c)